themes/tiddlywiki/vanilla/base.tid

```diff
@@ -333,16 +333,17 @@ button svg, button img {
 }
 
 .tc-btn-boxed {
-	font-size: 0.6em;
-	padding: 0.2em;
+	padding: 0 0.1em 0.1em 0.1em;
 	margin: 1px;
+	line-height: 1em;
 	background: none;
 	border: 1px solid <<colour tiddler-controls-foreground>>;
 	border-radius: 0.25em;
 }
 
 html body.tc-body .tc-btn-boxed svg {
-	font-size: 1.6666em;
+	margin-top:-0.1em;
```

Comment:
Which one do you mean? The deleted one or the added one?

A negative margin is nothing strange. The element wasn't centered and it wasn't asking for more padding. But yes, a font-size of 1.6666em does indeed lok somewhat strange.

The deleted one I introduced first in order to fix the wrong button sizes that were there beforehand ...when the buttons were styled as if they wouldn't come with text, too, and so the text suddenly was only 50% of what it should have been.

So, I fixed the button size (by reducing it) ...only increasing the svg.